### PR TITLE
Add Alex to Master

### PR DESCRIPTION
**Pull request by Martin0xFF**
- Added Hello_world.py
- Updated gitignore
- Added README.md

```bash
python hello_world.py
```

### CODE REVIEW

**Comment by Martin0xFF** [comment]
This pr is really cool and adds nice features

File: graham/hi.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("hi")
```
Comment (grahamhoyes) [inline graham/hi.py:1]: This is redundant, remove this file

**Review by grahamhoyes — CHANGES_REQUESTED** [review]

File: alex/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+scratch*.py
```
Comment (ChanHyukYang) [inline alex/.gitignore:1]: itchy

Comment (Martin0xFF) [inline alex/.gitignore:1]: Its a good idea to not commit these files, nice work!

Comment (Martin0xFF) [reply alex/.gitignore:1]: I second!

**Review by grahamhoyes — APPROVED** [review]
Looks good!

**Review by ChanHyukYang — APPROVED** [review]

**Pull request merged**